### ISSUE TITLE
refined_loss_1 is nan

### ISSUE THREAD

**Issue by fanglinpu** [vasgaowei/pytorch_MELM#7]
The refined_loss_1 is nan when I train the model using vgg16 backbone.

**Comment by vasgaowei** [comment]
Maybe the learning rate is larger. And you can try learning rate 0.001 which is the same as my settings'.

**Comment by fanglinpu** [comment]
I use the learning rate 0.001, but the refined_loss_1 is nan.

**Comment by WangBoying** [comment]
i have the same trouble

**Comment by WangBoying** [comment]
> I use the learning rate 0.001, but the refined_loss_1 is nan.

please，have you solved this problem?

**Comment by vasgaowei** [comment]
Make sure you download the right version of pretrained 'vgg16.pth' file. And I give the link of the version of pretrained 'vgg16.pth' file which woks fine.
